lib/node/fullnode.js

```diff
@@ -87,6 +87,7 @@ class FullNode extends Node {
       chain: this.chain,
       mempool: this.mempool,
       prefix: this.config.prefix,
+      agent: this.config.str('agent'),
```

Comment:
I'm using this `agent` string in the tests to exactly identity peers. I'm doing this to prevent the need to create custom identity keys.